.env.example

```diff
@@ -1,9 +1,15 @@
 DATABASE_DEV_URL = postgres://username:password@localhost:5432/database_name
 DATABASE_TEST_URL = postgres://username:password@localhost:5432/database_name
+
 SECRET_KEY = secretkey
+
 HOST_URL =
+LOCAL_URL = http://127.0.0.1:3000/api/v1
```

Comment:
we can remove this URl's as we are not using them anywhere in the codebase.

Reply:
Ok, I have done that, 
I have removed all unused environmental variables.